packages/ai-core/src/application-api.ts

```diff
@@ -5,19 +5,19 @@
  */
 import { OpenApiRequestBuilder } from '@sap-cloud-sdk/openapi';
 import type {
-  AllArgoCDApplicationData,
-  Body,
-  ArgoCDApplicationCreationResponse,
-  ArgoCDApplicationStatus,
-  ArgoCDApplicationData,
-  ArgoCDApplicationBaseData,
-  ArgoCDApplicationModificationResponse,
-  ArgoCDApplicationDeletionResponse,
-  ArgoCDApplicationRefreshResponse
+  BckndAllArgoCDApplicationData,
+  BckndArgoCDApplicationData,
+  BckndArgoCDApplicationDataRepoName,
+  BckndArgoCDApplicationCreationResponse,
+  BckndArgoCDApplicationStatus,
+  BckndArgoCDApplicationBaseData,
+  BckndArgoCDApplicationModificationResponse,
+  BckndArgoCDApplicationDeletionResponse,
+  BckndArgoCDApplicationRefreshResponse
 } from './schema/index.js';
 /**
  * Representation of the 'ApplicationApi'.
- * This API is part of the 'AI_CORE_API' service.
+ * This API is part of the 'src' service.
```

Comment:
Strange comment. This might come from one of the properties in the options-per-service.json, maybe packageName?

Reply:
Indeed, I'll play around with the options and check if we can improve it